geniza/corpus/tests/test_corpus_views.py

```diff
@@ -203,6 +203,24 @@ def test_pgp_metadata_for_old_site():
 
 
 class TestDocumentSearchView:
+    def test_document_suppression(self, document, suppressed_document):
+        SolrClient().update.index(
+            [
+                document.index_data(),  # no scholarship records
+                suppressed_document.index_data(),  # one scholarship record
+            ],
```

Comment:
You can actually just pass the documents, parasolr will check for and call the `index_data` method

Reply:
48387c8ea314293018515303bc09478485505a38 ✅

Reply:
It looks like there's a problem with indexing the objects:
```
TypeError: Object of type Document is not JSON serializable
```
Let me know if you'd like me to open a ticket on parasolr

Reply:
oh, my mistake — you're using the update index method and not the index items on the indexable class; I wasn't reading closely enough because I just somehow assumed you were doing what I would do, sorry 🤦‍♀️ 

You can use this `Document.index_items([document, suppressed_document])`

Reply:
aaaaand I was unable to get this to work either 😅 querying solr after this returned an empty queryset.